test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -360,7 +360,10 @@ TEST_P(ProxyProtocolTest, ErrorFIONREAD_1) {
                                 'r',  'e',  ' ',  'd',  'a',  't',  'a'};
   Api::MockOsSysCalls os_sys_calls;
   TestThreadsafeSingletonInjector<Api::OsSysCallsImpl> os_calls(&os_sys_calls);
-  EXPECT_CALL(os_sys_calls, ioctl(_, FIONREAD, _)).WillOnce(Return(Api::SysCallIntResult{-1, 0}));
+  // ioctl replaced with recv so have recv return an error
```

Comment:
nit: I don't think this comment adds much value, same below.

Reply:
Fixed and changed name of tests and comments.